miniml/compiler/test/Makefile

```diff
@@ -0,0 +1,37 @@
+OCAMLRUN_CMD=$(or $(OCAMLRUN),$(error $(ERRMSG)))
+define ERRMSG
+Fatal error:
+This Makefile depends on an OCAMLRUN variable pointing to an
+ocamlrun binary compatible with the bytecode produced by the
+compiler.
+
+A standard way to invoke this testsuite is by using the
+'test-compiler' target of the root makefile, which defines OCAMLRUN
+automatically, using the ocamlrun of ocaml-src in _boot/.
+
+If you have already run 'make ocamlrun' at the root, you may use the following:
+
+make OCAMLRUN=../../../_boot/byterun/ocamlrun
+
+
+endef
+
+
+COMPILE_CMD=GUILE_WARN_DEPRECATED=detailed guile ../compile.scm
```

Comment:
What is `detailed`?

Reply:
Oh, I hadn't read the `GUILE_WARN_DEPRECATED` part and I thought `detailed` was a command.